cli/rbd/client.py

```diff
@@ -0,0 +1,42 @@
+from copy import deepcopy
+
+from cli.utilities.utils import build_cmd_args
+
+
+class Client:
```

Comment:
To use `self.execute`, class needs to be inherited from `Cli`.
Refer - https://github.com/red-hat-storage/cephci/blob/master/cli/__init__.py